en/test_and_ci/README.md

```diff
@@ -2,22 +2,10 @@
 
 PX4 offers extensive unit testing and continuous integration facilities. This page provides an overview.
 
-## Testing on the local machine
+* [Unit Tests](../test_and_ci/unit_tests.md)
+* [Continuous Integration (CI)](../test_and_ci/continous_integration.md)
+* [Jenkins CI](../test_and_ci/jenkins_ci.md)
```

Comment:
In the sidebar this is under Continuous integration topic. Should we pull up a level in sidebar or indent here as well?

Reply:
I think indent here as well